### PR TITLE
Add audio modding article

### PR DESCRIPTION
**Pull request by milleroski**
Hi! I finally decided to port my audio modding guide over to this wiki! It extensively explains how someone can create an audio replacement mod using CustomSounds from the beginning to the end.

Please let me know if I should change anything -- I'm always open to suggestions :)

### CODE REVIEW

**Comment by MaxWasUnavailable** [comment]
We'll probably want to set up a custom category for mod-specific guides (= guides using other mods or APIs to implement something, rather than being completely standalone). We will merge this once we have that set up.

Matter of keeping the playing field fair for all APIs.

**Comment by milleroski** [comment]
Yeah, I totally agree with you there. It would make navigation for users easier as well since it's not immediately obvious that my guide uses CustomSounds -- that's why I wrote a note at the beginning in the first place.

**Comment by legoandmars** [comment]
Appreciate the PR a ton, should make helping people make audio mods much easier

Once #29 is merged in I'll look into setting up a mod-specific section so we can work on getting this in

File: docs/user-guide/modding/audio-modding-customsounds.md [hunk 1]
```diff
@@ -0,0 +1,132 @@
+# A guide to Lethal Company audio modding (using CustomSounds)
+
+?>**Note:** This is not a tutorial on how to replace audio with BepInEx using Harmony Patching or with LCSoundTool. This is a guide for using CustomSounds to replace audio. If you want to make a standalone audio mod without CustomSounds, this guide is not for you.
```
Comment (legoandmars) [inline docs/user-guide/modding/audio-modding-customsounds.md:3]:
```suggestion
::: warning 
This is not a tutorial on how to replace audio with BepInEx using Harmony Patching or with LCSoundTool. This is a guide for using CustomSounds to replace audio. If you want to make a standalone audio mod without CustomSounds, this guide is not for you.
:::
```

replace with new formatting

File: docs/user-guide/modding/audio-modding-customsounds.md [hunk 2]
```diff
@@ -0,0 +1,132 @@
...
+
+Afterwards, go to the directory you exported the audio files to, and go to Lethal Company\ExportedProject\Assets\AudioClip, and there, you should be able to see all the audio files that are currently in the game! Feel free to copy these files somewhere else or to delete the .meta files.
+
+You can usually find the files just by looking at the audio names, they have pretty logical names. For example, the ship "ice cream truck music" is literally called "IcecreamTruckV2" and "IcecreamTruckFar". 
+
+However, if you REALLY can't find the sound file you're looking for, then **Step 2.2** is for you. Otherwise, you can go directly to **Step 3**.
+
+### Using LCSoundTool
+
+?>**Note:** If you already have found the audio file you're looking for, you can skip directly to the next section.
```
Comment (legoandmars) [inline docs/user-guide/modding/audio-modding-customsounds.md:38]:
```suggestion
::: tip
If you already have found the audio file you're looking for, you can skip directly to the next section.
:::
```
replace with new formatting

File: docs/user-guide/modding/audio-modding-customsounds.md [hunk 3]
```diff
@@ -0,0 +1,132 @@
...
+- CustomSounds by Clementinise (<https://thunderstore.io/c/lethal-company/p/Clementinise/CustomSounds/>)
+
+If you're using r2modman or the Thunderstore manager, your basic setup should look like this:
+![Image of r2modman setup](../docs/files/audio-modding-customsounds/r2modman_setup.png)
+
+## Finding the audio files 
+If you want to replace an audio file, you need to know which sound you're actually replacing. For the sake of the guide, I'm going to try and find the audio files for the "Icecream Truck" music whenever the delivery ship lands, but you can apply this process to any sound you'd like to replace.
+
+### Extracting the audio files
+To find the sound file you want to replace, I personally recommend extracting all the audio files from the game so you have easy access to them, which makes finding them a lot easier. You can do this by using a program called AssetRipper (<https://github.com/AssetRipper/AssetRipper/releases>) or alternatively AssetStudio (<https://github.com/zhangjiequan/AssetStudio/releases>). Make sure to use the latest version (NOT the pre-release one).
```
Comment (legoandmars) [inline docs/user-guide/modding/audio-modding-customsounds.md:21]: I wouldn't recommend AssetStudio, it only supports up to 2022.3.1

Just AssetRipper should work fine

**Review by legoandmars — CHANGES_REQUESTED** [review]
Thanks a ton for the PR, the article looks good, we've just been undergoing a lot of formatting changes and the formatting needs to be slightly updated

- Any changes to sidebar.md and index.html can be discarded, as we're using a new structure now
- All images should be moved to `/docs/public/images/customsounds` with the in-guide links changed to `/images/customsounds/IMAGENAME`
- The page should  be moved to `/docs/apis/apis/customsounds.md`
- The page can be added to the sidebar through `docs/.vitepress/config.mts`, under the "Modding APIs" section (right under the page that says "Modding APIs Overview"

**Comment by milleroski** [comment]
Alright, I've adapted the guide to the new layout and added the changes you've requested, so everything should work fine now. I've tested it locally by running "npm run dev" and it looks fine, but it seems that the workflow is failing to configure the github pages. Specifically, I get the error "Create pages site failed". Any ideas as to why?

**Comment by legoandmars** [comment]
> Alright, I've adapted the guide to the new layout and added the changes you've requested, so everything should work fine now. I've tested it locally by running "npm run dev" and it looks fine, but it seems that the workflow is failing to configure the github pages. Specifically, I get the error "Create pages site failed". Any ideas as to why? Additionally, the guide should be probably referenced directly in the API overview page and given a gold star.

I haven't messed with how the github pages build works on forks, it's probably just not set up properly by default. I doubt it'll be an issue

**Review by legoandmars — APPROVED** [review]

**Pull request merged**